src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
```

Comment:
The class name 'Quadrangle' is appropriate as it doesn't use a verb and represents a geometric figure.